frontend/express/public/core/device-and-type/javascripts/countly.views.js

```diff
@@ -315,7 +315,7 @@ var TechnologyHomeWidget = countlyVue.views.create({
 
             var dd = [
                 {
-                    "title": CV.i18n('common.bar.top-platform'),
+                    "title": CV.i18n('common.bar.top-platform_'),
```

Comment:
We shouldn't add a new translation to make an existing one uppercase. So you should avoid this.

To make the strings uppercase, please add .text-uppercase class to the elements you desired. :slightly_smiling_face: 

I left separate comments for translation changes you've made.